src/plugins/projectexplorer/devicesupport/localprocesslist.cpp

```diff
@@ -86,7 +86,7 @@ QList<DeviceProcessItem> LocalProcessList::getLocalProcesses()
 
 #endif //Q_OS_WIN
 
-#ifdef Q_OS_UNIX
+#if defined(Q_OS_UNIX) || defined(Q_OS_OS2)
```

Comment:
There is Q_OS_UNIXLIKE:
```suggestion
#if defined(Q_OS_UNIXLIKE)
```